.github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml

```diff
@@ -0,0 +1,47 @@
+name: Feature Request
+description: A suggestion for a new feature
+labels: [enhancement]
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thank you for contributing to Swift-DocC-Render!
+      
+      Before you submit your issue, please replace each text area below with information about your proposed feature.
+- type: input
+  attributes:
+    label: Feature Name
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Description
+    description: |
+      A description of your proposed feature.
+
+      Design proposals to show what's missing, or what new capabilities will be possible, are very helpful! Provide links to existing issues or external references/discussions, if appropriate.
+
+      How does it behaves in different viewports?
```

Comment:
```suggestion
      How does it behave in different viewports?
```

Reply:
I'm not sure this is generally relevant for feature requests. This would only apply to specific kind of UI-related requests, so I'm not sure it should be included in the template.

Reply:
To me it's a question for the user who is writing the proposal to think about how the feature that wants to implement also works in different viewports. Does it make sense?

Reply:
Maybe we can also add something like "Are there accessibility considerations for this feature?"